prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -18,12 +18,9 @@ public class SelfDeclarationRule : AutomaticAdjudicationRule
     {
         public override Task<bool> ProcessRule(Enrollee enrollee)
         {
-            // check to see if any of the self-declaration rules were answered as 'Yes'
+            // check to see if the enrollee has any self declarations
             // note: if for some reason the question was not answered, we will assume 'Yes'
-            if (enrollee.HasConviction.GetValueOrDefault(true)
-                || enrollee.HasDisciplinaryAction.GetValueOrDefault(true)
-                || enrollee.HasPharmaNetSuspended.GetValueOrDefault(true)
-                || enrollee.HasRegistrationSuspended.GetValueOrDefault(true))
+            if (enrollee.SelfDeclarations != null && enrollee.SelfDeclarations.Any())
```

Comment:
Im pretty sure SelfDeclarations will be null only if we forget to .Include() the collection. Its probably bad if this rule passes if someone has self-declared and we accidentally remove the include. Null collection should probably also fail the rule.